### DISCUSSION TITLE
2013/12/03/%E4%BC%9A%E8%AF%B4%E8%AF%9D%E7%9A%84%E5%B0%8F%E8%B1%86

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#357] [Announcements]
# 2013/12/03/%E4%BC%9A%E8%AF%B4%E8%AF%9D%E7%9A%84%E5%B0%8F%E8%B1%86

豆豆还不到三岁，但是已经非常会说话了。尤其是豆豆已经非常清楚大人们喜欢听什么了，很多时候，他会有意的说一些大人喜欢听的话，即便那不是他自己的真实想法。所以，我每次听豆豆说他喜欢幼儿园，都不禁想，他这是在安慰我呢？还是真心喜欢幼儿园了呢？

https://blog.qizhen.xyz/2013/12/03/%E4%BC%9A%E8%AF%B4%E8%AF%9D%E7%9A%84%E5%B0%8F%E8%B1%86

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Arthur
2013年12月4日 08:33 编辑
真是可爱的小家伙啊。我也当爸爸一个多月了，最近开始学习你的育儿博客，挺有收获的，想起了很久以前研究你的labview文章做reentrant feature的日子:-)

回复
